src/platform/packages/shared/kbn-lens-embeddable-utils/config_builder/schema/color.ts

```diff
@@ -9,124 +9,92 @@
 
 import type { TypeOf } from '@kbn/config-schema';
 import { schema } from '@kbn/config-schema';
+import { isNil } from 'lodash';
 import { serializedValueSchema } from './serializedValue';
 
-const colorByValueBase = schema.object({
-  type: schema.literal('dynamic'), // Specifies that the color assignment is dynamic (by value). Possible value: 'dynamic'
+const colorByValueStepsSchema = schema.arrayOf(
+  schema.object({
+    /**
+     * The value from which this color applies (inclusive).
+     */
+    from: schema.maybe(
+      schema.nullable(
+        schema.number({
+          meta: { description: 'The value from which this color applies (inclusive).' },
+        })
+      )
+    ),
+    /**
+     * The value up to which this color applies (inclusive).
```

Comment:
Looking at the current UI all the steps except the last follow this logic `gte - lt`, so the `to` is exclusive and the form is inclusive. The last one is `gte - lte`. We should probably be explicit with that somehow

In the screenshot below the ranges are:
- `gte: 0 lt: 20`
- `gte: 20 lt: 80`
- `gte:80 lte: 100`


<img width="462" height="248" alt="Image" src="https://github.com/user-attachments/assets/4a8806e9-4842-4285-9105-1b9239e02466" />